Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.5.3
```

Comment:
I think you want `python:3.5-stretch`

Reply:
I don't recommend using any tags other than purely numeric ones (such as *-strech or *-alpine) and 3.5.3 is in fact outdated, but someone needs to test out 3.5.6 (or 3.5 since it's linked there) before we use that one.

Reply:
the production environment is stretch, not jessie
I am quite confident 3.5.6 will work (our travis runs tests on 3.7.0a4+)